### DISCUSSION TITLE
免费版重新升级成专业版

### DISCUSSION THREAD

**Discussion by Trucare1** [getlantern/lantern-forum-cn#365] [Q&A]
蓝灯闪退，重新安装软件。专业版变成免费版。如何才能重新升级成专业版？
![BNS NO 1290-4764](https://github.com/getlantern/lantern-forum-cn/assets/165857085/ebc7ba61-bcd9-4b92-9ddb-9bc502c90f68)

**Comment by MaxMKMax** [comment]
@Trucare1 您好! 非常抱歉因为系统上出了一些问题, 我们正在修复了. 
我们暂时会通过手动帮您升级或发送激活码. 请您提供我您的账号邮箱.
非常抱歉也谢谢您支持蓝灯!

**Reply by Trucare1** [reply]
2367375573@qq.com

**Comment by Trucare1** [comment]
付款通知时使用的邮箱是：med-exp@hotmail.com

**Reply by MaxMKMax** [reply]
@Trucare1 嗯嗯. 好的. 已经帮您提交上去了. 请留意邮件或者GitHub.

**Reply by MaxMKMax** [reply]
@Trucare1 您好. 已经给您发激活码了. 请查看邮件. 谢谢!
